frontend/benefit/handler/src/components/applicationList/ApplicationList.tsx

```diff
@@ -30,12 +30,13 @@ export interface ApplicationListProps {
 
 const buildApplicationUrl = (
   id: string,
-  status: APPLICATION_STATUSES
+  status: APPLICATION_STATUSES,
+  openDrawer = false
 ): string => {
   if (status === APPLICATION_STATUSES.DRAFT) {
     return `${ROUTES.APPLICATION_FORM}?id=${id}`;
   }
-  return `${ROUTES.APPLICATION}?id=${id}`;
+  return `${ROUTES.APPLICATION}?id=${id}${openDrawer ? '&openDrawer=1' : ''}`;
```

Comment:
Starts to feel a bit fragile. Also, in query params if the parameter exists, it can be considered true.

Reply:
Good catch! Resolving this with a function to check for param's value where it's used/needed.